scylla/src/transport/locator/mod.rs

```diff
@@ -3,10 +3,11 @@ pub mod replicas;
 pub mod replication_info;
 #[cfg(test)]
 mod test;
-pub mod token_ring;
+pub(crate) mod token_ring;
```

Comment:
`TokenRing` is a part of `locator::ReplicaLocator` public API and is re-exported a few lines down. I don't think there is a reason to keep the `token_ring` module as public (or `pub(crate)`).